src/main/java/com/puppycrawl/tools/checkstyle/api/LocalizedMessage.java

```diff
@@ -415,7 +416,11 @@ public int compareTo(LocalizedMessage other) {
 
         if (lineNo == other.lineNo) {
             if (columnNo == other.columnNo) {
-                result = getMessage().compareTo(other.getMessage());
+                result = Comparator.nullsFirst(Comparator.<String>naturalOrder())
```

Comment:
@romani 
> Classes which are missed in Cobertura coverage report:
com/puppycrawl/tools/checkstyle/api/LocalizedMessage

It looks like either `Comparator.nullsFirst` or `Comparator.naturalOrder` forces cobertura to ignore the entire class like we have seen with `streams`.

@Luolc Please rewrite this class not using those 2 methods and figure out which is causing the problem.